libnavigation-core/api/current.txt

```diff
@@ -678,11 +689,13 @@ package com.mapbox.navigation.core.routealternatives {
   }
 
   public final class AlternativeRouteMetadata {
+    method public int getAlternativeId();
```

Comment:
Wondering where this came from. Is this Copilot related?

Reply:
This is definitely not coming from Copilot. Wondering if it's a merge conflict issue or a Metalava one. Will check. Thanks for flagging!

Reply:
I see that removing both entries and running `$> make core-check-api` is still ✅ 

Will remove them. I believe the issue is https://github.com/mapbox/mapbox-navigation-android/issues/6257